components/support/nimbus-fml/src/workflows.rs

```diff
@@ -55,8 +55,8 @@ fn load_feature_manifest(
     channel: &str,
 ) -> Result<FeatureManifest> {
```

Comment:
Should we test coverage for recursive imports?

Reply:
I could be missing something, but I can't seem to figure out how `snake.yaml` and `00-head.yaml` are getting tested 😅 Should we add a couple of tests for them here, or are they included in existing tests?